### ISSUE TITLE
A+ only curve

### ISSUE THREAD

**Issue by pozsothereal** [ZsBT/hass-w1000-portal#27]
Hi, 

i have added only the +A curve, as it can be enough for me, but its not working without the 1.8.0 curve added to a report as well:

Ez a hiba egy egyéni integrációból lett származtatva.

Logger: custom_components.w1000-energy-monitor
Source: custom_components/w1000-energy-monitor/__init__.py:333
Integration: W1000 Energy Monitor (documentation, issues)
First occurred: 18:32:21 (2 occurrences)
Last logged: 18:32:21

1.8.0 curve is missing. Please add it to your report 'a60'
1.8.0 curve is missing. Please add it to your report 'b60'
![IMG_3050](https://github.com/ZsBT/hass-w1000-portal/assets/61325530/2f610eaf-a7c3-4338-91ff-10992b6c8f0b)

**Comment by ZsBT** [comment]
Igen, a jelenlegi verzió függ mindkét görbétől, mert együtt használja az óraállással (ami tényleg nem szükséges a fogyasztáshoz). E hibát is javítja majd az átírás, ami a többit is (pl. #21 ).
Üdv: Zsombor

**Comment by pozsothereal** [comment]
Szia, rendben, köszi előre is!
